src/components/Card/Card.js

```diff
@@ -1,30 +1,67 @@
 import { Tile } from "carbon-components-react";
 import React, { useState } from "react";
+import CardContent from "./CardContent/CardContent";
+import CardControls from "./CardControls/CardControls";
+import CardHeader from "./CardHeader/CardHeader";
 
-const Card = () => {
-  const [isChecked, setIsChecked] = useState(false);
+const Card = (props) => {
+  const [cardState, setCardState] = useState({
+    isChecked: false,
+    isEditMode: false,
+    ...props.contentState,
+  });
 
-  function checkBoxClickHandler() {
-    setIsChecked(!isChecked);
+  const currentValues = props.contentState;
+
+  function editHandler(data) {
+    setCardState((prevState) => {
+      return { ...prevState, ...data };
+    });
+  }
+
+  function saveButtonHandler(data) {
+    setCardState((prevState) => {
+      currentValues.content = prevState.content;
+      currentValues.header = prevState.header;
```

Comment:
What if you'll have more data object properties? You'll need to add them everywhere in the code. Think about the way to re-organize your card data object to get/update back the whole data object(via spread operator, for example).